docs/source/guide/release_notes/onprem/2.4.3.md

```diff
@@ -4,6 +4,8 @@ hide_sidebar: true
 
 ## Label Studio Enterprise 2.4.3
 
+<div class="onprem-highlight">New <code>splitchannels</code> option on audio configs, keyboard shortcuts for the Data Manager, restore locked annotations</div>
```

Comment:
potentially silly question - should we use backticks (`) instead of code tags around `splitchannels`, to avoid line breaks? The GH preview suggests we'll get potentially unwanted linebreaks from using the code tags

Reply:
I tried, but they didn't render correctly in the HTML. I think it's because they're in a <div> element and not a <p> element? It was either leave them unstylized or specify the HTML.

Reply:
SGTM, fine to leave as is!